tracing-subscriber/Cargo.toml

```
@@ -54,13 +54,16 @@ parking_lot = { version = ">= 0.7, <= 0.11", optional = true }
 
 # registry
 sharded-slab = { version = "^0.0.9", optional = true }
+thread_local = "1.0.1"
```

This dependency is only necessary when the `registry` feature flag is enabled. Can we add 
```suggestion
thread_local = { version = "1.0.1", optional = true }
```
and change this line
https://github.com/tokio-rs/tracing/blob/7ad6826208421d09808eeb81b9667c9aa0c4a0a3/tracing-subscriber/Cargo.toml#L30
to
```toml
registry = ["sharded-slab", "thread-local"]